src/App.tsx

```diff
@@ -1,11 +1,13 @@
 import './App.scss';
 
 import SignUp from 'components/Signup';
+import { NavigationBar } from 'components/NavigationBar';
 
 function App() {
```

Comment:
I know that isn't part of your PR but you can change it on arrow fuction and export like NavigationBar

Reply:
I mean the App component

Reply:
fixed